src/Util/Functions.js

```diff
@@ -167,4 +167,29 @@ module.exports = {
+        emojis.map((e) => {
```

Comment:
This would be better with an `<Array>.forEach()` instead. In fact, why not just add the emojis to the alphabet string? That would save time spent pushing the emojis into the `abc` variable.

Reply:
It's not much time spent on it and I got the array after making the ABC var so I just push it

Reply:
Fair enough then.

Reply:
Nope, map should be used